updateTLDs.sh

```diff
@@ -1,9 +0,0 @@
-# Script to update the list of gTLDs
-curl -o data/iana_gtlds.txt http://data.iana.org/TLD/tlds-alpha-by-domain.txt
-echo "ONION" >> data/iana_gtlds.txt
```

Comment:
Oops! I forgot to account for this one manual TLD addition.

Probably this should just be special cased to have a `delegationDate` == when the CABF blessed issuance for the EV certs for `.onion` domains? Open to suggestions on how to handle this case. I think it could also benefit from a unit test so that it isn't forgotten in the future.